apps/src/craft/code-connection/CraftVisualizationColumn.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {default as GameButtons} from '../../templates/GameButtons';
+import BelowVisualization from '../../templates/BelowVisualization';
+import ProtectedVisualizationDiv from '../../templates/ProtectedVisualizationDiv';
+
+export default React.createClass({
```

Comment:
Have we opened the flood gates to people using es6 classes for react components yet? I think the answer was yes? @Bjvanminnen?